### PR TITLE
Readme: Referencing the wiki when building for linux

### PR DESCRIPTION
**Pull request by maxdunbar**
When building dolphin, the wiki has lots of helpful information for dependencies. I am proposing to add a direct link to that page on the wiki under the building for linux section on the README

### CODE REVIEW

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

**Review by Pokechu22 — APPROVED** [review]

File: Readme.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ since the Binary directory contains a working Dolphin distribution.
 Dolphin requires [CMake](https://cmake.org/) for systems other than Windows. Many libraries are
 bundled with Dolphin and used if they're not installed on your system. CMake
 will inform you if a bundled library is used or if you need to install any
-missing packages yourself.
+missing packages yourself. You may refer to the [Wiki](https://wiki.dolphin-emu.org/index.php?title=Building_Dolphin_on_Linux) for more information.
```
Comment (PEmu2) [inline Readme.md:64]: "Wiki" probably doesn't need to be capitalized here.

**Comment by cobalt2727** [comment]
I would suggest looking at linking to https://github.com/dolphin-emu/dolphin/wiki/Building-for-Linux as well - pretty much anyone can edit it, but it gets updates faster than the official wiki.

**Comment by maxdunbar** [comment]
> I would suggest looking at linking to https://github.com/dolphin-emu/dolphin/wiki/Building-for-Linux as well - pretty much anyone can edit it, but it gets updates faster than the official wiki.

Is one preferred? It seems like both have the same purpose.

**Comment by maxdunbar** [comment]
> Wiki maintainer here, I personally wouldn't mind linking the build instructions from the GitHub wiki instead of the Dolphin wiki, I think it's actually better to concentrate all development-related info where the source code actually resides. 
> 
> 
> 
> In fact, the Linux build instructions from the Dolphin wiki are often outdated and it goes a long time like that due to the more restricted nature of that wiki (register an account on the wiki, then register an account on the forums to ask for wiki edit permissions and then wait a little to get them)...

Great to hear a concrete response, I am currently waiting for permission to edit the Dolphin wiki. I am hearing that I should provide a link to both wikis. However, I worry that could cause confusion on which wiki to use. Thoughts?

**Comment by mbc07** [comment]
I personally would link only the page from the GitHub wiki.  At this point, the dedicated Linux build instructions page in Dolphin wiki is there mostly due to historical reasons and because it's still referenced at some other places...

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

**Review by phire — APPROVED** [review]

**Pull request merged**